test/board_test.rb

```diff
@@ -40,11 +40,20 @@ def test_valid_placement
     assert_equal false, @board.valid_placement?(@cruiser, ["A1", "B2", "C3"])
   end
 
-  def test_can_place_horizontal
+  def test_can_valid_horizontal_coordinates
     assert_equal true, @board.valid_placement?(@cruiser, ["A1", "A2", "A3"])
   end
 
-  def test_can_place_vertical
+  def test_can_valid_vertical_coordinates
```

Comment:
The change from place to valid makes a lot of sense. This is a great name change for the method!